docs/autogenerated-menu/New Content/3 career/people_ops/talent/recruitment_process_overview.md

```diff
@@ -0,0 +1,156 @@
+
+- **Past** – Our history, our roots, and how we got to where we are today.
+- **Present** – Who we work with, what we do, and how we operate.
+- **Future** – Where we’re heading and how they could be part of it.
+- **Human Connection** – Do they communicate well? Do they seem like a good cultural fit?
+- **Aspirations** – Does what they want align with what we can offer?
+
+By the end of this call, we should have a sense of whether they’d be a good fit—and they should be excited about the opportunity to join us.
+
+---
+
+## Candidate Review: What We Look For
+
+Beyond technical skills, we look for:
+- **Willingness to Learn** – No one knows everything, but we value curiosity and growth.
+- **Polyglot Mindset** – A broad range of knowledge is an asset.
+- **Positive Attitude** – Aptitude can be taught, attitude cannot.
+- **Comfort with Challenge** – We like people who embrace the unknown.
+
+If a candidate meets these criteria, they move forward to interviews.
+
+---
+
+## Interviews: Assessing Fit and Capability
+
+### Recruitment Rota: A Scalable Approach
+
+Interviews at Burendo are **structured and fair**. Our recruitment rota ensures:
+- **A mix of interviewers** to avoid bias.
+- **Consistency in hiring decisions.**
+- **A structured training approach** for new interviewers.
+
+### Interview Best Practices
+- **Set the Scene** – Explain what the interview involves.
+- **Sell Burendo** – Make them want to work here.
+- **Make Them Comfortable** – It should be a conversation, not an interrogation.
+- **Ask the Right Questions** – Cover both skills and behaviours.
+- **Encourage Dialogue** – Let them ask us questions too.
+
+---
+
+## Values Interview: Culture Fit
+
+We assess how well candidates align with our values:
+1. **One Team** – They should be great collaborators.
+2. **Quality** – They should take pride in delivering great work.
+3. **Continuous Improvement** – They should always be learning.
+4. **Integrity** – They should do the right thing, even when things go wrong.
+5. **Empathy** – They should care about their colleagues and clients.
+
+---
+
+## Skills Test: Hands-On Ability
+
+We believe the best way to assess skills is **by doing**. That’s why we set a **practical, role-specific test** that allows candidates to demonstrate their creativity and capability.
+
+---
+
+## Skills Interview: Beyond Technical Knowledge
+
+We don’t just look at what candidates *know*—we look at how they *think* and *work*:
+- **Problem-Solving** – How they approach challenges.
+- **Delivery Approach** – How they turn ideas into reality.
+- **Leadership** – How they manage their work and relationships.
+- **Consultancy Mindset** – How they adapt to new environments.
+
+---
+
+## The Offer: Making It Official
+
+Once a candidate has passed all stages, the **Talent Team and Board** handle the offer. The offer is shaped by interview feedback, and we ensure it’s competitive and fair.

```

Comment:
I think you can still pass &| fail stages and get an offer or not.  Not sure we should shape the interview as a box tick process, but an organic one.